lib/pg_lock/locket.rb

```diff
@@ -18,13 +18,13 @@ def unlock
     end
 
     def aquired?
-      @lock[0]["pg_try_advisory_lock"] == "t"
+      [true, "t"].include?(@lock[0]["pg_try_advisory_lock"])
```

Comment:
It's up to you. If you have the time, i'll merge it. There is another case where you used the same Array.